### PR TITLE
Artifactory HA 6.10.3. Update installer-info template

### PR DESCRIPTION
**Pull request by eldada**
#### PR Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [x] Chart Version bumped
- [x] CHANGELOG.md updated
- [x] Variables and other changes are documented in the README.md

**What this PR does / why we need it**:
Artifactory 6.10.3.
Update installer-info template.

### CODE REVIEW

File: stable/artifactory-ha/templates/artifactory-installer-info.yaml [hunk 1]
```diff
@@ -10,16 +10,13 @@ metadata:
...
       "features": [
       {
-        "featureId": "AppVersion/{{ default .Chart.AppVersion .Values.artifactory.image.version }}"
+        "featureId": "ArtifactoryVersion/{{ default .Chart.AppVersion .Values.artifactory.image.version }}"
       },
       {
-        "featureId": "Installer/{{ default "helm" .Values.installer.type }}"
-      },
-      {
-        "featureId": "Platform/{{ default "kubernetes" .Values.installer.platform }}"
```
Comment (jainishshah17) [inline stable/artifactory-ha/templates/artifactory-installer-info.yaml]: Keep this as its used in different fork of chart repo for other platforms like GKE-MP, Rancher, IBM

Comment (jainishshah17) [reply stable/artifactory-ha/templates/artifactory-installer-info.yaml]: @ankushchadha Can give you more context.

Comment (eldada) [reply stable/artifactory-ha/templates/artifactory-installer-info.yaml]: These changes were requested by @freddy33 .
@freddy33 - do you want to comment here?

Comment (jainishshah17) [reply stable/artifactory-ha/templates/artifactory-installer-info.yaml]: one of the example -> https://github.com/IBM/charts/blob/master/community/artifactory-ha/values.yaml#L11

Comment (eldada) [reply stable/artifactory-ha/templates/artifactory-installer-info.yaml]: Ok. `Platform` I'll add again. Do we really have use for the `Installer/`?

Comment (jainishshah17) [reply stable/artifactory-ha/templates/artifactory-installer-info.yaml]: That you can remove.

**Pull request merged**